src/steps/step_list.go

```diff
@@ -66,18 +66,17 @@ type WrapOptions struct {
 // Wrap wraps the list with steps that
 // change to the Git root directory or stash away open changes.
 func (stepList *StepList) Wrap(options WrapOptions, repo *git.ProdRepo) error {
-	previousBranch, err := repo.Silent.PreviouslyCheckedOutBranch()
-	if err != nil {
-		return err
+	previousBranch, errPrev := repo.Silent.PreviouslyCheckedOutBranch()
+	currentBranch, errCurr := repo.Silent.CurrentBranch()
```

Comment:
Tiny optimization, should we only run this if we have a previous branch? Then may not need the special err names.

Reply:
Good idea! https://github.com/git-town/git-town/pull/1600 resolves this.